docs/network/bridge/README.md

```diff
@@ -1,40 +1,57 @@
 # Network Bridge Device Plugin
 
-This device plugin allows user to attach pods to bridges running on nodes.
+This device plugin allows user to attach pods to existing bridges on nodes.
 
 ## Usage
 
-User must specify list of bridges that should be exposed to pods. Please note,
-that bridges must be manually precreated on all nodes.
+The user starts by ensuring that a bridg eexists on a given host.
+If it doesn't exit, then he can easily create it as a root as follows:
 
+```bash
+$ ip link create mybr0 type bridge
+$ ip link set dev mybr0 up
+$ ip link show mybr0
+6: mybr0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/ether fe:1b:e5:82:d3:b5 brd ff:ff:ff:ff:ff:ff
 ```
-kubectl create configmap device-plugin-network-bridge --from-literal=bridges="br0,br1"
+
+Now the user must specify list of bridges that should be exposed to pods:
```

Comment:
could be worth saying that multiple bridges can be passed as "br1,br2"

Reply:
done